src/utils/datetime/formatters.ts

```diff
@@ -177,6 +177,35 @@ export const createDateTimeFormatter = (
       }
     }
 
+    case 'YYYY-MM-DD': {
+      const options = {
+        ...dateTimeOptions,
+        hour12: false,
+      }
+
+      if (timeZone === 'UTC') {
+        options.timeZone = 'UTC'
+      }
+      const formatter = Intl.DateTimeFormat('en-us', options)
+
+      const formatDate = date => {
+        const parts = formatter.formatToParts(date)
+        const dateParts: any = {}
+
+        parts
+          .filter(part => part.type !== 'literal')
+          .forEach(part => {
+            dateParts[part.type] = part.value
+          })
+
+        return `${dateParts.year}-${dateParts.month}-${dateParts.day}`
+      }
+
+      return {
+        format: formatDate,
+      }
+    }
+
```

Comment:
We needed a formatter case and here is why:

I was getting a bunch of e2e test failures and couldn't understand why, for a long time. I thought it was just flakiness, but when I looked at the artifacts, I noticed that the new Formatter was not handling the `YYYY-MM-DD` case.

The cypress would `type` : `2019-09-02T08:00:00Z` but the text field would read `2019-09-02 12:00:00 AM T08:00:00Z`. I traced that down and turns out we were not handling the `YYYY-MM-DD` case so that upon a valid case of `YYYY-MM-DD`, the formatter didn't error out.